app/controllers/application_controller.rb

```diff
@@ -114,7 +114,15 @@ def load_admin_banner
     if Rails.env.development?
       @admin_banner = AdminBanner.where(:active => true).last
     else
-      @admin_banner = Rails.cache.fetch("admin_banner"){AdminBanner.where(:active => true).last}
+      # http://stackoverflow.com/questions/12891790/will-returning-a-nil-value-from-a-block-passed-to-rails-cache-fetch-clear-it
+      # Basically we need to store a nil separately.
+      @admin_banner = Rails.cache.fetch("admin_banner") do 
+        banner=AdminBanner.where(:active => true).last
+        banner.nil? ?  "" : banner
```

Comment:
Unnecessary spacing detected.<br>Operator `?` should be surrounded by a single space.